object/mailing_list.go

```diff
@@ -47,6 +47,15 @@ func (n Node) SyncFromGoogleGroup() {
 		if AuthorMember == nil {
 			continue
 		}
+
+		for msgIndex, msgItem := range messages {
+			if msgItem.Files != nil {
+				for _, msgFileItem := range msgItem.Files {
+					messages[msgIndex].Content = fmt.Sprintf("%s<br>[%s](%s)", messages[msgIndex].Content, msgFileItem.FileName, msgFileItem.Url)
```

Comment:
I think using `msgItem` is more elegant than `messages[msgIndex]`.

Reply:
At line 54, each file link will be appended at the end of the messages[msgindex].Content

Reply:
or like this?
```go
for msgIndex, msgItem := range messages {
	if msgItem.Files != nil {
		for _, msgFileItem := range msgItem.Files {
			msgItem.Content = fmt.Sprintf("%s<br>[%s](%s)", msgItem.Content, msgFileItem.FileName, msgFileItem.Url)
		}
		messages[msgIndex].Content = msgItem.Content
	}
}
```

Reply:
@WindSpiritSR Oh, sorry, I forgot that this is not a pointer array, then these codes are ok. Please squash your 2 commits then this PR will be ok.

Reply:
@Kininaru It’s okay, I made the same mistake when I wrote it ahhhh
Already squashed